tools/python/util/convert_onnx_models_to_ort.py

```diff
@@ -258,24 +259,33 @@ def parse_args():
         "processed.",
     )
 
-    return parser.parse_args()
+    parsed_args = parser.parse_args()
+    parsed_args.optimization_style = [OptimizationStyle[style_str] for style_str in parsed_args.optimization_style]
+    return parsed_args
 
 
-def convert_onnx_models_to_ort():
-    args = parse_args()
+def convert_onnx_models_to_ort(
+    model_path_or_dir: pathlib.Path,
+    output_dir: pathlib.Path | None = None,
+    optimizations: list[OptimizationStyle] | None = None,
```

Comment:
could we name it `optimization_styles`? I think `optimizations` has a different meaning.